client/syntaxes/coq.tmLanguage

```diff
@@ -45,7 +45,7 @@
 
         <dict>
             <key>match</key>
-            <string>(Lemma|Theorem|Goal|Example)\s+([a-zA-Z][a-zA-Z0-9_']*)</string>
+            <string>(Lemma|Theorem|Goal|Example)\s+(\p{L}(\p{L}|[0-9]|[_'])+)</string>
```

Comment:
And why do you exclude one-letter identifiers?

Reply:
Thanks again; fixed.

Reply:
Also, you can keep `[0-9]` and `[_']` merged, as in the original code. (sorry for the multiple comments, I just noticed this)